### PR TITLE
Verify ocs and spoke sync

### PR DESCRIPTION
**Pull request by alknopfler**
- verify in ocs step
- verify for registry sync into spokes: verify, verify_ocp_sync and verify_olm_sync

### CODE REVIEW

File: deploy-ocs/verify.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+# Load common vars
+source ${WORKDIR}/shared-utils/common.sh
+if [[ -z ${ALLSPOKES} ]]; then
+	ALLSPOKES=$(yq e '(.spokes[] | keys)[]' ${SPOKES_FILE})
+fi
+
+for spoke in ${ALLSPOKES}; do
+	echo ">>>> Deploy manifests to install LSO and LocalVolume: ${spoke}"
```
Comment (jparrill) [inline deploy-ocs/verify.sh:25]: Change the message to match a verification :D

File: deploy-ocs/verify.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+	echo "Extract Kubeconfig for ${spoke}"
+	extract_kubeconfig ${spoke}
+	if [[ $(oc get --kubeconfig=${SPOKE_KUBECONFIG} pod -n openshift-storage | grep -i running | wc -l) -ne $(oc --kubeconfig=${SPOKE_KUBECONFIG} get pod -n openshift-storage --no-headers | grep -v Completed| wc -l) ]]; then
+
```
Comment (jparrill) [inline deploy-ocs/verify.sh:30]: We need to add some verifications here:
- For LSO: PVs and Running Pods
- For OCS: StorageClasses available and Running pods

**Review by jparrill — CHANGES_REQUESTED** [review]

**Review by alknopfler — COMMENTED** [review]
LGTM

**Review by jparrill — APPROVED** [review]

**Comment by jparrill** [comment]
LGTM

**Pull request merged**